src/comparator_tests/boolean_test.js

```diff
@@ -0,0 +1,76 @@
+var QUnit = require("steal-qunit");
+
+var set = require('src/set-core'),
```

Comment:
@jandjorgensen just an FYI, this isn't a good way to import things.  You should use relative paths or include the name of the module like 'can-set/set-core'.